backend/protzilla/methods/data_analysis.py

```diff
@@ -395,28 +693,117 @@ class DifferentialExpressionKruskalWallisOnIntensity(DataAnalysisStep):
         "corrected_alpha",
     ]
 
-    calc_method = staticmethod(kruskal_wallis_test_on_intensity_data)
-
-    def insert_dataframes(self, steps: StepManager, inputs) -> dict:
-        inputs["protein_df"] = steps.get_step_output(Step, "protein_df", inputs["protein_df"])
-        inputs["metadata_df"] = steps.metadata_df
-        inputs["log_base"] = steps.get_step_input(TransformationLog, "log_base")
-        return inputs
-
-
-class DifferentialExpressionKruskalWallisOnPTM(DataAnalysisStep):
-    display_name = "Kruskal-Wallis Test"
-    operation = "Peptide analysis"
-    method_description = ("A function to conduct a Kruskal-Wallis test between groups defined in the clinical data."
-                          "The p-values are corrected for multiple testing.")
-
-    output_keys = [
-        "differentially_expressed_ptm_df",
-        "significant_ptm_df",
-        "corrected_p_values_df",
+    def create_form(self):
+        return Form(
+            label="Kruskal-Wallis Test",
+            input_fields=[
+                DropdownField(name="protein_df", label="Step to use protein data from"),
+                DropdownField(
+                    name="multiple_testing_correction_method",
+                    label="Multiple testing correction",
+                    options=MultipleTestingCorrectionMethod,
+                    value=MultipleTestingCorrectionMethod.benjamini_hochberg,
+                ),
+                FloatField(
+                    name="alpha",
+                    label="Error rate (alpha)",
+                    value=0.05,
+                    min=0,
+                    max=1,
+                    step=0.01,
+                    separatePrefix="\u03B1",
+                ),
+                DropdownField(name="grouping", label="Grouping from metadata"),
+                DropdownField(
```

Comment:
Use `MultiSelectField` instead of `DropdownField` here, because the backend expects a list rather than a single string.

```suggestion
               MultiSelectField(
```

Reply:
You are right. Is fixed now, thanks :))